### ISSUE TITLE
src/maindialog.h is missing copyright holder info

### ISSUE THREAD

**Issue by ArrayBolt3** [lxqt/obconf-qt#201]
##### Expected Behavior
All files containing a copyright header should identify the copyright holder.

##### Current Behavior
src/maindialog.h currently contains the following data at the top of the copyright header:
    <one line to give the program's name and a brief idea of what it does.>
    Copyright (C) 2013  <copyright holder> <email>

##### Possible Solution
Pretty sure this is Hong Jen Yee (PCMan)'s copyright? Many of the other files in the src directory are under his copyright in the year 2013.

##### Steps to Reproduce (for bugs)
Open src/maindialog.h in GitHub using Falkon web browser. Problem is visible in master and in 0.16.2.

##### Context
I am updating the copyright file for the Ubuntu packaging for obconf-qt, and this is throwing a wrench in the works. Since I can't be certain who owns the copyright, I'm not sure who to copyright it as. Just knowing who owns the copyright should be enough AFAIK, but as it is, I'm about to have a line in debian/copyright that reads `Copyright: 2013      <copyright holder> <email>`.

##### System Information
Probably irrelevant, but just in case/for comic value:
* Distribution & Version: Lubuntu 22.10 Kinetic Kudu Alpha
* Kernel: 5.15.0-27-generic
* Qt Version: 5.15.4
* libqtxdg Version: 3.9.1-1
* lxqt-build-tools Version: 0.11.0-0.1
* Package version: 0.16.2

**Comment by zhrexl** [comment]
Yep. The fact that maindialog.cpp has his copyright strongly indicates maindialog.h is also his work and copyright.
